core/src/main/scala/akka/persistence/postgres/config/AkkaPersistenceConfig.scala

```diff
@@ -82,14 +83,24 @@ class SnapshotPluginConfig(config: Config) {
   override def toString: String = s"SnapshotPluginConfig($dao)"
 }
 
+class TagsConfig(config: Config) {
+  private val cfg = config.asConfig("tags")
+  val cacheTtl: FiniteDuration = cfg.asFiniteDuration("cacheTtl", 1.hour)
+  val insertionRetryAttempts: Int =
+    if (cfg.hasPath("insertionRetryAttempts")) cfg.getInt("insertionRetryAttempts")
+    else 1
```

Comment:
`config.asInt("insertionRetryAttempts", 1)`  is probably what you want

Reply:
That's what I was looking for! Thanks 😊